frontend/lib/src/components/elements/Video/Video.test.tsx

```diff
@@ -143,7 +149,13 @@ describe("Video Element", () => {
         startTime: 10,
       })
       render(<Video {...props} />)
-      expect(document.body).toMatchSnapshot()
+      const videoElement = screen.getByTestId("stVideo")
+      expect(videoElement).toBeInTheDocument()
```

Comment:
nit/question: does this line provide any benefit considering the next line would fail if it didn't exist?

Reply:
Good point 👍 Doing the equivalent in playwright can have some advantages, but in RTL there is probably no real benefit of doing that -> I removed it

Reply:
for my knowledge, what benefits would that have in playwright?

Reply:
In playwright, a root cause for flakiness can be the unpredictability of how long some browser interaction/loading will take. E.g., an element might be added to DOM after a couple seconds and take another couple seconds to load some content in the element. 

Doing more fine-grained steps - e.g. 1. check if the element is in DOM 2. check if the content is correct - will give more room for some unpredictable slowness at one of the steps without slowing anything down.

Also, [actions in playwright](https://playwright.dev/docs/actionability) can have different degrees of waiting "actionability checks". For some of these actions, it is recommended to use waiting checks before to make sure that an element is in the targeted state. E.g., for `assert_snapshot`, it's good to always have some checks beforehand to make sure that all the relevant elements are loaded.

Reply:
makes sense. thanks for the explanation!